lib/Game.js

```diff
@@ -1,9 +1,10 @@
 class Game {
-  constructor(player1, player2, player3, round = 1) {
+  constructor(players, round = 1) {
     this.players = [];
     this.currentRound = round;
     this.puzzle = new Puzzle();
     this.currentPuzzle = this.puzzle.puzzleArray[round-1];
+    this.currentWheel = null;
```

Comment:
Is there a reason we don't generate the wheel right away? When does `createNewWheel` get called?